ts/Stock/Indicators/VBP/VBPIndicator.ts

```diff
@@ -34,11 +41,8 @@ const {
     noop
 } = H;
 import SeriesRegistry from '../../../Core/Series/SeriesRegistry.js';
-const {
-    seriesTypes: {
-        sma: SMAIndicator
-    }
-} = SeriesRegistry;
+const SMAIndicator: typeof SMAIndicatorType = SeriesRegistry.seriesTypes.sma;
+const columnPrototype = SeriesRegistry.seriesTypes.column.prototype;
```

Comment:
Shouldn't be:
```
import type ColumnSeriesType from '../../../Series/Column/ColumnSeries';
....
const columnProto: ColumnSeriesType = SeriesRegistry.seriesTypes.column.prototype;
```
Just like in other indicators (AOIndicator, MACDIndicator). 

Or maybe the other way around. `const columnPrototype = SeriesRegistry.seriesTypes.column.prototype;` is perfectly fine, and we can simply replace `const SMAIndicator: typeof SMAIndicatorType = SeriesRegistry.seriesTypes.sma;` with `const SMAIndicator = SeriesRegistry.seriesTypes.sma;`? Tested on RSIIndicator, seems to be fine

Reply:
Right, I was thinking about that too and then decided to be rather consistent than mix deconstruction and regular assignment. In the produced codes deconstructions become regular assignments, so we do not produce overhead. Also some changes are temporary until we can produce native declarations and then have the assembler replaced to use regular named imports everywhere and remove all the import const and namespaces.

Reply:
Okay, thanks!